doc/ranger.pod

```diff
@@ -209,6 +209,9 @@ This does not work over ssh, requires certain terminals (tested on "xterm" and
 To enable this feature, install the program "w3m" and set the option
 C<preview_images_method> to w3m.
 
+When using a terminal with a border larger than 0 which is not automatically detected the w3m preview will be misaligned.
```

Comment:
Add a comma after detected. And are there terminals whose border gets detected automatically? If none are detected we need a comma before which. And I like "nonzero border" better than "border larger than 0," presumably there could be terminals with negative borders and if not it's still not wrong.

Reply:
Auto detected is maybe not the right wording for it. But what I mean is that `w3mimgdisplay -test` returns different values between `st` and `urxvt` with the same border px and geometry which is what causes the misalignment. So technically `urxvt`'s padding gets 'auto detected'.  

What would you recommend changing it into?

Reply:
Auto detected because the terminal reports something different is fine imo. You could go with something more generic like "properly handled" but I'm not sure that's better.

Reply:
I'll make it like this then `When using a terminal with a nonzero border, which is not automatically detected, the w3m preview will be misaligned.`